lib/product.rb

```diff
@@ -0,0 +1,17 @@
+class Product
```

Comment:
It looks like you are missing the method implementation for `hoard` and `is_hoarded?` here, please read the readme a bit more closely next time 🙈